docs/content.md

```diff
@@ -535,6 +535,58 @@ view = api.content.get_view(
 %
 % self.assertEqual(view.__name__, u'plone')
 
+(content-get-path-example)=
+
+## Get content path
+
+To get the path of a content object, use {meth}`api.content.get_path`.
+This method returns either an absolute path from the Zope root or a relative path from the portal root.
+
+The following example shows how to get the absolute path from the Zope root
+
+```python
+from plone import api
+portal = api.portal.get()
+
+folder = portal['events']['training']
+path = api.content.get_path(obj=folder)
+assert path == '/plone/events/training'
+```
+
+The following example shows how to get the portal-relative path.
+
+```python
+rel_path = api.content.get_path(obj=folder, relative=True)
+assert rel_path == 'events/training'
+```
+
+If the API is used to fetch an object outside the portal with the {meth}`relative` parameter set as `True`, it throws a `InvalidParameterError`.
```

Comment:
Using `meth` for `relative` looks wrong. @stevepiercy, what should we use here?

Reply:
It's a parameter of the method, not a method itself. Also this restructured sentence is more clear.

```suggestion
If the API is used to fetch an object with the `relative` parameter set as `True`, and  the object is outside the portal, it throws an `InvalidParameterError` error.
```